src/Custom/Chat/ChatClient.cs

```diff
@@ -14,6 +15,7 @@ namespace OpenAI.Chat;
 public partial class ChatClient
 {
     private readonly string _model;
+    private readonly InstrumentationFactory _instrumentation;
```

Comment:
Would it be acceptable to call the type "OpenTelemetrySource" and the field "_telemetry"?